tests/unit/test_apigateway.py

```diff
@@ -560,6 +561,21 @@ def test_render_custom_template(self, template):
             "stageVariable2": "value2",
         }
 
+    def test_render_valid_booleans_in_json(self):
```

Comment:
Similar to https://github.com/localstack/localstack/blob/c7d0336a3eb2f81b7b97af33a2ad135d74489bbe/tests/unit/test_templating.py#L254